benchmarks/base.py

```diff
@@ -18,6 +19,18 @@ def setup(self):
 
     def teardown(self):
         os.chdir(self.cwd)
+
+        if hasattr(self, "repo"):
+            self.repo.close()
+
+        if os.name == "nt":
+            # Windows does not allow to remove read only files
+            for root, _, files in os.walk(self.test_directory.name):
+                for file in files:
+                    path = f"{root}{os.sep}{file}"
+                    perm = os.stat(path).st_mode | stat.S_IWRITE
+                    os.chmod(path, perm)
+
         self.test_directory.cleanup()
```

Comment:
Hm, why this isn't enough? It seems to chmod on perm error by itself https://github.com/python/cpython/blob/master/Lib/tempfile.py#L794

Reply:
I have been receiving WinError 5 on my win machine, and included this code because of that, before I realized that errors for this PR have a different source. Let's see whether it actually causes problems here.

Reply:
Well, seems like it will be required
https://github.com/iterative/dvc-bench/runs/714891458?check_suite_focus=true#step:6:5237

I will reintroduce that tomorrow. Also, I will add writing run logs to file and upload alongside results

Reply:
Strange, but ok. Would leave a comment/create an issue, if it isn't clear why `cleanup` doesn't work.